requirements.txt

```diff
@@ -3,3 +3,5 @@ hvplot
 xarray
 netcdf4
 cartopy
+geoviews
+metpy
```

Comment:
I imagine this has heavy requirements. Can it be treated like cartopy, only used if it's available?

Reply:
I have handled this.